server/src/main/java/org/elasticsearch/persistent/PersistentTasksCustomMetaData.java

```diff
@@ -189,7 +190,7 @@ public long getNumberOfTasksOnNode(String nodeId, String taskName) {
 
     @Override
     public Version getMinimalSupportedVersion() {
-        return Version.V_5_4_0;
+        return Version.V_6_3_0;
```

Comment:
We cannot change this. This would mean that a mixed 6.2 x-pack / 6.3 x-pack cluster might drop its persistent tasks on the floor.
Instead I suggest to add another method to custom that says something like  `featureLessSince` which returns an optional version (default is Optional.empty()).
We can then override this method for PersistentTasksCustomMetaData to return Version.V_6_3_0.
Finally we'll make `shouldSerializeCustom` aware of this new `featureLessSince` method.

Reply:
Discussed with @bleskes. Only revert this to `return Version.V_5_4_0;`, the rest should be covered by other PRs

Reply:
I pushed a18f16628f6045dd57e0b3e5dce790b8afd623f4.